.circleci/config.yml

```diff
@@ -94,6 +77,7 @@ workflows:
           app_catalog_test: "control-plane-test-catalog"
           chart: "aws-operator"
           requires:
+            - push-aws-operator-to-aliyun
             - push-aws-operator-to-quay
```

Comment:
I'm not sure if we should add it to dependencies. From the correctness standpoint it should be there. Otherwise we shouldn't deploy to china but I'm uncertain how much it will impact build times.

Reply:
that should be fine, its not like the push takes hour its usually around 10min

Reply:
Then it totally sucks when you want to test things from your branch, no?

Reply:
Hmm, that's true for testing branches it delays the catalog  CI job by some time

Reply:
I checked few last builds from CI:
* push to quay CI job takes around 1-2mins 
* push to China takes 2-5 mins on average

so on average, we delay app catalog job by 3 minutes

app catalog CI job takes around 1 minute 

Its probabbly not that big issue in both cases (if we wait for china or not)

Reply:
Okay. Let's try that then.